public_html/includes/classes/forms.php

```diff
@@ -1137,7 +1137,15 @@ public static function buildEditTable($formID) {
 						);
 
 				foreach ($form['fields'] as $field) {
-					$temp[] = sprintf('<input type="%s" style="%s" name="%s_%s" value="%s" readonly />',
+          // checking if type is text then we will use textarea instead of an input
+					if ($field['type'] === 'text') {
```

Comment:
This change will have breaking side effects in the system. The side effects will range from cosmetic, to incorrect semantics, to inflexibility and could lead to issues with how it was planned to handle Linked data, once that is happening.